app/test_main.py

```diff
@@ -1,3 +1,63 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+class TestGetHumanAge:
+    @pytest.mark.parametrize(
+        "cat_age, dog_age, expected",
+        [
+            pytest.param(
+                0,
+                0,
+                [0, 0],
+                id="Human age is zero when pets age are zero",
+            ),
+            pytest.param(
+                14,
+                14,
+                [0, 0],
+                id="Human age is zero when pets age < 15"
+            ),
```

Comment:
You check here that 14 gives  0 in the result, but do you check that  15 gives  1 in the result?)
add more tests to check all age boundaries